lte/gateway/Makefile

```diff
@@ -211,7 +211,7 @@ ifdef UT_PATH
 endif
 	make -C $(MAGMA_ROOT)/lte/gateway/python unit_tests MAGMA_SERVICE=$(MAGMA_SERVICE) UT_PATH=$(ut_path) DONT_BUILD_ENV=$(DONT_BUILD_ENV)
 
-test_oai: ## Run all OAI-specific tests
+test_oai: stop ## Run all OAI-specific tests
```

Comment:
This will not work in CI as it doesn't have a concept of running services with systemd in that environment. Is this necessary? If it's useful to have a target that stops the running services and execute unit tests, we can create a separate target for this. :)